web/src/admin/users/UserListPage.ts

```diff
@@ -40,6 +41,56 @@ import {
     UserPath,
 } from "@goauthentik/api";
 
+export const requestRecoveryLink = (user: User) =>
```

Comment:
Breaking these two bits of functionality out so that they can be consistent between the two pages.